DigitalLearningSolutions.Data.Migrations/202204071000_AddDetailsLastCheckedToUsersTableAndRenameOnDelegateAccounts.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202204071000)]
+    public class AddDetailsLastCheckedToUsersTableAndRenameOnDelegateAccounts : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("Users").AddColumn("DetailsLastChecked").AsDateTime().Nullable();
+            Rename.Column("DetailsLastChecked").OnTable("DelegateAccounts").To("CentreSpecificDetailsLastChecked");

```

Comment:
This has been intentionally renamed to avoid confusion with the new one on Users. There was a Slack conversation about this, it just hasn't been documented on the ticket (I'll sort this).